gatsby-config.js

```diff
@@ -58,7 +58,13 @@ const plugins = [
         {
           resolve: 'gatsby-remark-prismjs',
           options: {
-            noInlineHighlight: true
+            noInlineHighlight: true,
+            languageExtensions: [
+              {
+                language: 'text',
```

Comment:
on what specific blocks do we get this error? should they be `dvc`?

Reply:
Here's a few I found:

docs/command-reference/install.md

    ## Disable Git hooks
    
    When you use `dvc install`, it creates three files under the `.git/hooks`
    directory:
    
    ```
    .git/hooks
    ├── post-checkout
    ├── pre-commit
    └── pre-push
    ```

content/docs/command-reference/repro.md

    ## Examples
    
    For simplicity, let's build a pipeline defined below. (If you want get your
    hands-on something more real, see this short
    [pipeline tutorial](/doc/start/data-pipelines)). It takes this `text.txt` file:
    
    ```
    dvc
    1231
    is
    3
    the
    best
    ```

content/docs/command-reference/pull.md

    ## Description
    
    [...]
    
    It has the same effect as running `dvc fetch` and `dvc checkout`:
    
    ```
    Controlled files             Commands
    ---------------- ---------------------------------
    
    remote storage
         +
         |         +------------+
         | - - - - | dvc fetch  | ++
         v         +------------+   +   +----------+
    project's cache                  ++ | dvc pull |
         +         +------------+   +   +----------+
         | - - - - |dvc checkout| ++
         |         +------------+
         v
     workspace
    ```

From what I can tell, each usage is for visual examples made in monospace ASCII and text files that really don't map to a language.

Reply:
yeah, I see - they are indeed generic. Thanks @rogermparent !